src/components/Widget/WidgetDialogContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import { ForwardedRef, forwardRef } from 'react'
+import { createPortal } from 'react-dom'
+import styled from 'styled-components/macro'
+import { Z_INDEX } from 'theme/zIndex'
+
+const StyledDialogContainer = styled.div<{ visible: boolean }>`
+  background-color: ${({ theme }) => theme.backgroundScrim};
+  opacity: ${({ visible }) => (visible ? 1 : 0)};
+  transition: visibility 0.25s linear, opacity 0.25s;
+  visibility: ${({ visible }) => (visible ? 'visible' : 'hidden')};
+
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
```

Comment:
might need to be `fixed`
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/39385577/219111615-9464f125-0e9f-41cc-927f-f66b9ec7d792.png">